src/Microsoft.PowerShell.Commands.Utility/resources/WebCmdletStrings.resx

```diff
@@ -126,9 +126,12 @@
   <data name="CredentialConflict" xml:space="preserve">
     <value>The cmdlet cannot run because the following conflicting parameters are specified: Credential and UseDefaultCredentials. Specify either Credential or UseDefaultCredentials, then retry.</value>
   </data>
-  <data name="DirecotryPathSpecified" xml:space="preserve">
+  <data name="DirectoryPathSpecified" xml:space="preserve">
     <value>Path '{0}' resolves to a directory. Specify a path including a file name, and then retry the command.</value>
   </data>
+  <data name="EmptyKeyInJsonString" xml:space="preserve">
+    <value>The provided json includes a property whose name is an empty string, this is only supported using the -AsHashTable switch.</value>
```

Comment:
Should probably capitalize json as `JSON`

Reply:
OK. Fixed it.